docs/helm-chart-submission.md

```diff
@@ -0,0 +1,21 @@
+# Submission of Helm Charts for verification
+
+OpenShift Helm Charts repository hosts Helm Charts that are available by default with OpenShift. You can use this repository to submit the charts that need to be certified through a Pull Request. Once your Pull Request is merged, a CI/CD pipeline is created which publishes the chart in the GitHub releases which is further reflected on the Helm charts [repository index](http://charts.openshift.io/).
+
+The submission process of a Helm Chart for Red Hat Helm Repository and Certification has been documented on the [Red Hat Helm Repository](https://github.com/openshift-helm-charts/charts). Please note the instructions mentioned on the repository before submitting a chart.
+
+The follow options are available for submitting a Chart for inclusion in Red Hat Helm Repository and Certification: 
```

Comment:
"The follow**ing** options are available...

Reply:
Done!